jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -658,9 +658,14 @@ IGN_PHYSICS_DEPENDENCIES="libbenchmark-dev \\
                           libignition-common3-dev \\
                           libignition-math6-dev \\
                           libignition-math6-eigen3-dev \\
-                          libignition-plugin-dev \\
-                          libsdformat9-dev \\
-                          libsdformat8-dev"
+                          libignition-plugin-dev"
+if [[ -n "${IGN_PHYSICS_MAJOR_VERSION}" && ${IGN_PHYSICS_MAJOR_VERSION} -ge 2 ]]; then
+  IGN_PHYSICS_DEPENDENCIES="${IGN_PHYSICS_DEPENDENCIES} \\
+                            libsdformat9-dev"
+elif [[ -n "${IGN_PHYSICS_MAJOR_VERSION}" && ${IGN_PHYSICS_MAJOR_VERSION} -ge 1 ]]; then
```

Comment:
If I'm not wrong, the if logic before leaves this comparison only valid against 1
```suggestion
elif [[ -n "${IGN_PHYSICS_MAJOR_VERSION}" && ${IGN_PHYSICS_MAJOR_VERSION} -eq 1 ]]; then
```